CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ### Changed
 
-- Scrape expiry of Secrets of type `kubernetes.io/tls`. ([#92](https://github.com/giantswarm/cert-exporter/pull/92))
+- Add metric which tracks expiry of TLS certificates stored in Kubernetes secrets. ([#92](https://github.com/giantswarm/cert-exporter/pull/92))
```

Comment:
Nice. If `cert_exporter_secret_not_after` is the metric that got added, let's name it here.

```suggestion
- Add metric `cert_exporter_secret_not_after` which tracks expiry of TLS certificates stored in Kubernetes secrets. ([#92](https://github.com/giantswarm/cert-exporter/pull/92))
```

Reply:
good call - done.